src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/2.sql

```diff
@@ -865,16 +865,27 @@ AS
 
     DECLARE @deletedInstances AS TABLE
         (StudyInstanceUid VARCHAR(64),
-         SeriesInstanceUid VARCHAR(64),
-         SopInstanceUid VARCHAR(64),
-         Status TINYINT,
-         Watermark BIGINT)
+            SeriesInstanceUid VARCHAR(64),
+            SopInstanceUid VARCHAR(64),
+            Status TINYINT,
+            Watermark BIGINT)
 
     DECLARE @studyKey BIGINT
+    DECLARE @seriesKey BIGINT
+    DECLARE @instanceKey BIGINT
     DECLARE @deletedDate DATETIME2 = SYSUTCDATETIME()
 
-    -- Get the study PK
-    SELECT  @studyKey = StudyKey
+    -- Get the study, series and instance PK
+    SELECT  @studyKey = StudyKey, @seriesKey =
```

Comment:
nit: formatting. null is keyword and needs tp CAPs. each select field should be one line. Helps with readability

 SELECT  @studyKey = StudyKey, 
         @seriesKey = CASE @seriesInstanceUid WHEN NULL THEN NULL ELSE SeriesKey END,
         @instanceKey = CASE @sopInstanceUid WHEN NULL THEN NULL ELSE InstanceKey END